modules/configuring-monitoring-for-the-multi-model-serving-platform.adoc

```diff
@@ -4,7 +4,9 @@
 = Configuring monitoring for the multi-model serving platform
 
 [role="_abstract"]
-The multi-model serving platform includes metrics for the ModelMesh component. When you have configured monitoring, you can grant Prometheus access to scrape the available metrics.
+The multi-model serving platform includes per model and model server metrics for the ModelMesh component. ModelMesh generates its own metrics and does not rely on the underlying serving runtime to provide them. The set of available metrics include metrics for model request rates and timings, model loading and unloading rates, times and sizes, internal queuing delays, capacity and usage, cache state, and least recently used models. For more information, see link:https://github.com/kserve/modelmesh-serving/blob/main/docs/monitoring.md[ModelMesh metrics]. 
+
+When you have configured monitoring, you can view available metrics for the ModelMesh component. 
```

Comment:
Per [this comment](https://github.com/opendatahub-io/opendatahub-documentation/pull/255#discussion_r1572255864), I believe you must still use Prometheus to view the metrics, it's just that you don't need to explicitly grant it access to do so. 

If Vedant has confirmed that is the case, then I would say "When you have configured monitoring, you can use Prometheus to view available metrics for the ModelMesh component".